index.html

```diff
@@ -164,23 +164,21 @@ <h3 class="work down">Work</h3>
               </li>
             </ul>
           </div>
-          </div>
-
-
-                
-        
+        </div>
         </section>
+
        
-        <div class="gray">
+       <div id="popup-project" class="popup-project"></div>
```

Comment:
[ OPTIONAL BUT RECOMMENDED] Pop-up should be in the center of the page both for small and large viewports. Please see the screenshots below and fix the issues.
![pop-up-desktop](https://user-images.githubusercontent.com/35504149/132742453-fbf26031-bf2e-470d-9266-9198a59e5154.png)
![pop-up-small](https://user-images.githubusercontent.com/35504149/132742495-2680ff59-67ce-458b-b193-4ef15c9d074c.png)